Danki2/Assets/Scripts/Camera/CustomCamera.cs

```diff
@@ -25,10 +41,22 @@ private void UpdateRotation()
     private void FollowTarget()
     {
         float zDistanceFromFloorIntersect = height / (Mathf.Tan(Mathf.Deg2Rad * angle));
-        float newX = target.transform.position.x;
-        float newY = target.transform.position.y + height;
-        float newZ = target.transform.position.z - zDistanceFromFloorIntersect;
+        var targetPosition = target.transform.position;
 
-        transform.position = new Vector3(newX, newY, newZ);
+        var mouseOffsetH = Input.mousePosition.x / Screen.width - 0.5f;
+        var mouseOffsetV = Input.mousePosition.y / Screen.height - 0.5f;
+
+        desiredPosition.Set(
+            targetPosition.x + mouseOffsetH * mouseFollowFactor,
+            targetPosition.y + height,
+            targetPosition.z - zDistanceFromFloorIntersect + mouseOffsetV * mouseFollowFactor
+        );
+
+        transform.position = Vector3.Lerp(transform.position, desiredPosition, 1f - smoothFactor);
```

Comment:
Should this Lerp be affected by Time.deltaTime somehow? I think at the moment, this could be different at different frame rates

Reply:
I had a play with this. I think the fact that this lerp is a geometric progression makes putting delta time in more complex and less useful.

Reply:
Yeah I think you're right, I had a go too and it just boggled me